packages/frontend/src/components/AllBids/SettledBidsList.tsx

```diff
@@ -39,6 +45,7 @@ export const SettledBidsList = ({ search }: SettledBidsListProps) => {
       ) : (
         <>
           <BidsListHeaders />
+          <GoldenTicketWinner bidderAddress={goldenTicketWinner?.bidderAddress} />
```

Comment:
Right now GoldenTicketWinner is displayed in golden ticket winner and raffle section. Let's remove it from raffle section.